pkg_resources/__init__.py

```diff
@@ -2098,8 +2098,13 @@ def _handle_ns(packageName, path_item):
     # capture warnings due to #1111
     with warnings.catch_warnings():
```

Comment:
Do we still need this warning with `importer.find_spec`, or does `find_spec` not trigger the behavior from #1111?

Reply:
@jaraco could you comment? I was never affected by #1111 and have not reproduced it. My intent with this change was minimal impact to existing behavior.

Reply:
I tried disabling this warning and did not see any ill effects. I do not know if this needs warning-kill needs to be kept, but I narrowed its scope in the latest commit.